_data/simple-icons.json

```
@@ -13744,6 +13744,11 @@
             "source": "https://www.sonarqube.org/logos/",
             "guidelines": "https://www.sonarqube.org/logos/"
         },
+        {
+            "title": "sonarr",
+            "hex": "2596BE",
+            "source": "https://sonarr.tv"
```

```suggestion
            "source": "https://github.com/Sonarr/Sonarr/blob/913b845faadc3c9fc005abfba815426743d01bdf/Logo/Sonarr.svg"
```

As with your other PR, I couldn't find the icon at the source - but I _could_ find it in their git repository.